custom/icds_reports/tasks.py

```diff
@@ -58,3 +64,52 @@ def move_ucr_data_into_aggregation_tables(date=None):
                 sql_to_execute = sql_file.read()
                 cursor.execute(sql_to_execute, {"date": date.strftime('%Y-%m-%d')})
             celery_task_logger.info("Ended icds reports update_daily_aggregate_table")
+
+
+@periodic_task(run_every=crontab(minute=0, hour=21), acks_late=True, queue='background_queue')
+def recalculate_stagnant_cases():
+    domain = 'icds-cas'
+    config_ids = [
+        'static-icds-cas-static-ccs_record_cases_monthly',
```

Comment:
@sheelio you originally said we only need to look at 4 UCRs, but I think we need 6 because of the v1/v2 for child cases and ccs records right?

Reply:
Yup that's right (though at some point we should deprecate the v1 data sources)